src/components/Header/HeaderAccountBalance.jsx

```diff
@@ -1,88 +1,135 @@
-import {
-  Grid,
-  Container,
-  Divider,
-  Stack,
-  Typography,
-  ListItemIcon,
-} from '@mui/material';
+import { Container, Box, Divider, Stack, Typography } from '@mui/material';
 import { HistoryOutlined, CalendarTodayOutlined } from '@mui/icons-material';
-import { flexbox } from '@mui/system';
 
-const accountWidth = 240;
+const AccountBalance = (props) => {
+  const filterAccountBalance = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'ISSUED' || current.status === 'OVERDUE'
+        ? prev + current.amount
+        : prev;
+    }, 0);
+  };
+
+  const invoiceOverdue = () => {
+    return props.data.reduce((prev, current) => {
+      return current.status === 'OVERDUE' ? prev + current.amount : prev;
+    }, 0);
+  };
+
+  const calculateDate = () => {
```

Comment:
This naming is misleading, this function does not calculate anything, it simply filters out a particular element of an array.
```suggestion
  const getLatestIssuedInvoice = () => {
```